python/cudf/cudf/core/_internals/copying.py

```diff
@@ -36,7 +37,7 @@ def gather(
 
 @acquire_spill_lock()
 def scatter(
-    sources: list[ColumnBase | cudf.Scalar],
+    sources: list[ColumnBase | ScalarLike],
```

Comment:
Why this change? I think `ScalarLike` is an alias for `Any`.

Reply:
If it remains `ScalarLike`, should we handle the different scalars we can get?

Reply:
Thanks. I changed this when trying to going down the rabbit hole of using `pylibcudf.Scalar` here but it became more involved that I had thought. This was a leftover I forgot to revert